test/commands/contents.test.js

```diff
@@ -74,7 +75,66 @@ describe('contents command handler', () => {
 		);
 	});
 
-	test('<search> value not found, logs info message in console error', async () => {
+	test('when 3 repos are tested and a limit of 2 is specified, only 2 repos are logged', async () => {
```

Comment:
You could also do these 3 tests as a [jest table](https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout)